.github/workflows/mac-tests.yml

```diff
@@ -22,7 +21,7 @@ jobs:
     runs-on: macos-latest
 
     # Not intended for forks.
-    if: github.repository == 'optuna/optuna'
+    if: (github.event_name == 'schedule' && github.repository == 'optuna/optuna') || (github.event_name != 'schedule')
```

Comment:
I think the above comment is not consistent with this condition since non-scheduled workflows are expected to be executed in fork repositories. We can see a similar change in L74.

Reply:
I update the comment.

Reply:
The same conditions appear on Tests.yml (and other tests). Is it better to add the comment to them?

Reply:
Thank you for your update. I think we should work on `Tests.yml` in a follow-up PR to minimize the changes in this PR.